GW 5.0.15 release notes (#5596)
TykTechnologies/tyk-docs: tyk-docs/content/product-stack/tyk-gateway/release-notes/version-5.0.md

---

## 5.0.15 Release Notes {#rn-v5.0.15}

### Release Date 15 October 2024

### Breaking Changes

There are no breaking changes in this release.

### Upgrade Instructions

Go to the [Upgrading Tyk](https://tyk.io/docs/product-stack/tyk-gateway/release-notes/version-5.0/#upgrading-tyk) section for detailed upgrade instructions.

### Release Highlights

This patch release for Tyk Gateway addresses critical stability issues for users running Tyk Gateway within the data plane, connecting to the control plane or Tyk Hybrid. Affected users should upgrade immediately to version 5.0.15 to avoid service interruptions and ensure reliable operations with the control plane or Tyk Hybrid.

For a comprehensive list of changes, please refer to the detailed [changelog]({{< ref "#Changelog-v5.0.15">}}) below.

### Changelog {#Changelog-v5.0.15}

#### Fixed

<ul>
<li>
<details>
<summary>Resolved gateway panic on reconnecting to MDCB control plane or Tyk Cloud</summary>

> In prior versions, Tyk Gateway could encounter panic when attempting to reconnect to the control plane after it was restarted. This patch version has resolved this issue, ensuring stable connectivity between the gateway and control plane following reconnections and reducing the need for manual intervention.

Reviewer: ```suggestion
In version 5.0.14, Tyk Gateway could encounter panic when attempting to reconnect to the control plane after it was restarted. This patch version has resolved this issue, ensuring stable connectivity between the gateway and control plane following reconnections and reducing the need for manual intervention.
```

---

## 5.0.15 Release Notes {#rn-v5.0.15}

### Release Date 15 October 2024

### Breaking Changes

> **Attention:** Please read this section carefully.

Reviewer 2: ```suggestion
```

Reviewer 2: @sharadregoti please remove it from the other prs. If there are no changes we don't need this warning

There are no breaking changes in this release.

### Upgrade Instructions

Go to the [Upgrading Tyk](https://tyk.io/docs/product-stack/tyk-gateway/release-notes/version-5.0/#upgrading-tyk) section for detailed upgrade instructions.

### Release Highlights

> This patch release for Tyk Gateway addresses critical stability issues for users running Tyk Gateway within the data plane, connecting to the control plane or Tyk Hybrid. Affected users should upgrade immediately to version 5.6.1 to avoid service interruptions and ensure reliable operations with the control plane or Tyk Hybrid.

Reviewer 2: why `Affected users should upgrade immediately to version 5.6.1`? we are in 5.0.15

Reviewer 3: That's a typo, I think.

> This patch release for Tyk Gateway addresses critical stability issues for users running Tyk Gateway within the data plane, connecting to the control plane or Tyk Hybrid. Affected users should upgrade immediately to version 5.6.1 to avoid service interruptions and ensure reliable operations with the control plane or Tyk Hybrid.

Reviewer 3: ```suggestion
This patch release for Tyk Gateway addresses critical stability issues for users running Tyk Gateway within the data plane, connecting to the control plane or Tyk Hybrid. Affected users should upgrade immediately to version 5.0.15 to avoid service interruptions and ensure reliable operations with the control plane or Tyk Hybrid.
```

For a comprehensive list of changes, please refer to the detailed [changelog]({{< ref "#Changelog-v5.0.15">}}) below.

### Changelog {#Changelog-v5.0.15}

#### Fixed

<ul>
<li>
<details>
<summary>Resolved gateway panic on reconnecting to MDCB control plane or Tyk Cloud</summary>

> In prior versions, Tyk Gateway could encounter a panic when attempting to reconnect to the control plane after it was restarted. This issue has been resolved in patch version 5.0.15, ensuring stable connectivity between the gateway and control plane following reconnections and reducing the need for manual intervention.

Reviewer 2: ```suggestion
In prior versions, Tyk Gateway could encounter panic when attempting to reconnect to the control plane after it was restarted. This patch version has resolved this issue, ensuring stable connectivity between the gateway and control plane following reconnections and reducing the need for manual intervention.
```

Reviewer 2: @sharadregoti please update the other prs

---

## 5.0.15 Release Notes {#rn-v5.0.15}

> ### Release Date 15 October 2024

Reviewer: ```suggestion
### Release Date 24 October 2024
```
